sdk/automanage/Azure.ResourceManager.Automanage/src/Generated/ConfigurationProfileVersionResource.cs

```diff
@@ -0,0 +1,430 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.Automanage
+{
+    /// <summary>
+    /// A Class representing a ConfigurationProfileVersion along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="ConfigurationProfileVersionResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetConfigurationProfileVersionResource method.
+    /// Otherwise you can get one from its parent resource <see cref="ConfigurationProfileResource" /> using the GetConfigurationProfileVersion method.
+    /// </summary>
+    public partial class ConfigurationProfileVersionResource : ArmResource
```

Comment:
each resource should have CRUD tests associated with it

Reply:
Used directives to exclude some of the operations that are not quite customer ready.